warp10/src/main/java/io/warp10/continuum/sensision/SensisionConstants.java

```diff
@@ -316,6 +316,11 @@
   public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS = "warp.standalone.delete.datapoints";
 
   /**
+   * Number of datapoints deleted by 'standalone' delete per owner and application
+   */
+  public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS_PEROWNER = "warp.standalone.delete.datapoints.perowner";
```

Comment:
Then simply rename the metric to 'perownerapp', You can then REDUCE by 'owner' or 'app'.
If cardinality ever becomes a problem we can always add a TTL to the metric.